modules/shortcodes/vimeo.php

```diff
@@ -78,7 +78,22 @@ function vimeo_shortcode( $atts ) {
 		$height = round( ( $width / 640 ) * 360 );
 	}
 	
+	/**
+	 * Filter the Vimeo player width.
+	 *
+	 * @since 3.4-beta1
```

Comment:
Alrighty, updated the tag to use a stable release. Since core uses a trailing `.0` I did as well.